pyfstat/core.py

```diff
@@ -286,12 +286,17 @@ def __init__(
         injectSources : dict or str
             Either a dictionary of the values to inject, or a string pointing
             to the .cff file to inject
-        injectSqrtSX :
-            Per-IFO single-sided PSD values for generating fake Gaussian noise on the fly
-        assumeSqrtSX : float
-            Don't estimate noise-floors but assume (stationary) per-IFO
-            sqrt{SX} (if single value: use for all IFOs). If signal only,
-            set sqrtSX=1
+        injectSqrtSX : float or list or str
+            Single-sided PSD values for generating fake Gaussian noise on the fly.
+            Single float or str value: use same for all IFOs.
+            List or comma-separated string: must match len(detectors)
+            and/or the data in sftfilepattern.
+        assumeSqrtSX : float or list or str
+            Don't estimate noise-floors but assume this (stationary) single-sided PSD.
+            Single float or str value: use same for all IFOs.
+            List or comma-separated string: must match len(detectors)
+            and/or the data in sftfilepattern.
```

Comment:
Same proposal:
```suggestion
            and/or the data in sftfilepattern. Detectors will be paired to list elements following alphabetical order.
```

So the difference is `injectSqrtSX` adds Gaussian noise to the data and estimates PSD from the data itself, while `assumeSqrtSX` just says "PSD value is this one."?

Reply:
I think/hope the estimation is completely independent of this option, one should be able to use it with/without `assumeSqrtSX".